### PR TITLE
Homepage top slice

### PR DESCRIPTION
**Pull request by asavin**
giphy 1

### Motivation
https://zpl.io/Z1HlVhw

### Test plan
- Check how top slice on homepage looks like on all the browsers and devices

### Pre-flight checklist
- [x] Has been design reviewed
- [x] Add a gif - `gif`

### CODE REVIEW

File: site/pages/home/homepage-top-slice/style.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+.homepageTopSlice {
+  padding: 50px 60px 100px 60px;
+  text-align: center;
+  background-color: white;
+}
+
+.badgerSlogan {
+  composes: fontXL from "../../../css/typography/_fonts.css";
+  font-weight: bold;
```
Comment (zoeabryant) [inline site/pages/home/homepage-top-slice/style.css]: is font-weight bold not a feature of fontXL on purpose? same for fontM on line 17 - can you check with the designers about the reusability of these styles?

File: site/pages/home/homepage-top-slice/style.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+@media largeScreen {
+  .homepageTopSlice {
+    text-align: center;
+    padding: 160px 0 230px 0;
+  }
+
+  .sliceContainer {
+    max-width: 730px;
```
Comment (zoeabryant) [inline site/pages/home/homepage-top-slice/style.css]: is this max-width exclusive to the homepage top slice?

Comment (asavin) [inline site/pages/home/homepage-top-slice/style.css]: yup that's specific to that slice

File: site/pages/home/homepage-top-slice/style.css (hunk 1 above)
Comment (asavin) [inline site/pages/home/homepage-top-slice/style.css]: This is actually irrelevant now, removed font-weight properties.

File: site/css/typography/_fonts.css [hunk 3]
```diff
@@ -56,18 +56,20 @@
...
   font-weight: 800;
 }
 
 .fontL, .fontM {
   font-family: "Sentinel A", "Sentinel B", Georgia, serif;
   font-size: 24px;
-  line-height: 1.1em;
+  line-height: 1.2em;
   font-style: normal;
   font-weight: 800;
```
Comment (zoeabryant) [inline site/css/typography/_fonts.css]: 👍

**Pull request merged**